src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,59 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int NAME_POS = 0;
+    private static final int NUM_POS = 1;
+    private int buySum;
+    private int supplySum;
+    private int result;
```

Comment:
looks like it is not needed to create these variables on class level, move its to the methods where its called.